features/nvstore/source/nvstore.h

```diff
@@ -50,6 +50,11 @@ typedef enum {
 // defines 2 areas - active and nonactive, not configurable
 #define NVSTORE_NUM_AREAS        2
 
+/** NVStore class
+ *
+ *  NVStore Class
```

Comment:
> NVStore Class

why is this doubled here?

Reply:
@SenRamakri @AnotherButler Care to answer?

Reply:
I would expect here something similar to https://github.com/ARMmbed/mbed-os/blob/master/platform/FileBase.h#L36, even an example how to use this object (some drivers have it).

Reply:
@davidsaada - Can you please add some description of what that class is? For example, see - https://github.com/ARMmbed/mbed-os/blob/master/features/cellular/easy_cellular/EasyCellularConnection.h#L30